versioned_docs/version-8.7/self-managed/identity/miscellaneous/configuration-variables.md

```diff
@@ -144,6 +144,84 @@ Setting either of the feature flags to `true` requires a database connection. To
+| `MULTITENANCY_ENABLED` | Controls the multi-tenancy feature within Identity. This can be set in helm via `camunda.tasklist.multi-tenancy.enabled`. | `false`       |
+
+With multi-tenancy enabled, the following variables can be used to configure additional [tenants](/self-managed/identity/user-guide/tenants/managing-tenants.md). To add additional tenants, increment the `0` value for each variable (for example, `IDENTITY_TENANTS_1_NAME`).
+
+| Environment variable                         | Description                                                                                                                                                                                                                                        | Default value |
+| -------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `IDENTITY_TENANTS_0_NAME`                    | A human-readable name for the tenant.                                                                                                                                                                                                              | Default       |
+| `IDENTITY_TENANTS_0_TENANTID`                | The tenant ID. The `<default>` tenant is automatically created during Identity startup.                                                                                                                                                            | `<default>`   |
+| `IDENTITY_TENANTS_0_MEMBERS_0_TYPE`          | The type of member to add to the tenant, one of `GROUP`, `USER`, or `APPLICATION`. Additional members can be added by incrementing the second `0` value (for example, `IDENTITY_TENANTS_0_MEMBERS_1_TYPE`.)                                        | `APPLICATION` |
+| `IDENTITY_TENANTS_0_MEMBERS_0_APPLICATIONID` | If adding an **application** as a member of the tenant, the application ID. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_APPLICATIONID`). | `zeebe`       |
+| `IDENTITY_TENANTS_0_MEMBERS_0_USERNAME`      | If adding a **user** as a member of the tenant, the username. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_USERNAME`).                    | `user-name`   |
+| `IDENTITY_TENANTS_0_MEMBERS_0_GROUPNAME`     | If adding a **group** as a member of the tenant, the name of the group. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_GROUPNAME`).         | `group-name`  |
```

Comment:
@romansmirnov 

> ❓ Assigning a member (user, group, or application) to a tenant is only possible when using Keycloak. In other words, these configuration properties do not apply when integrating/connecting to an OIDC provider (without Keycloak).

> In the docs, do we have a concept that reflects the different environments (with Keycloak, without Keycloak, but with an OIDC integration provider) that Identity can run in? Depending on it, some properties apply and others not.

I am not confident we do. Is there a portion of this you would remove until we can more thoroughly focus on this topic in the backlog?